include/flash_map_pm.h

```diff
@@ -11,11 +11,11 @@
 #include <sys/util.h>
 
 /* Aliases for zephyr - mcuboot/ncs style naming */
-#define image_0 mcuboot_primary
-#define image_1 mcuboot_secondary
-#define image_0_nonsecure mcuboot_primary
-#define image_1_nonsecure mcuboot_secondary
-#define image_scratch mcuboot_scratch
+#define image_0 MCUBOOT_PRIMARY
+#define image_1 MCUBOOT_SECONDARY
+#define image_0_nonsecure MCUBOOT_PRIMARY
+#define image_1_nonsecure MCUBOOT_SECONDARY
+#define image_scratch MCUBOOT_SCRATCH
 
 #if (CONFIG_SETTINGS_FCB || CONFIG_SETTINGS_NVS)
 #define storage settings_storage
```

Comment:
SETTINGS_STORAGE, LITTLEFS_STORAGE, NVS_STORAGE

Reply:
Ah, true, will look into it